### ISSUE TITLE
CVE on glob-parent

### ISSUE THREAD

**Issue by dlbock** [cypress-io/cypress-browserify-preprocessor#87]
### Is this a Feature or Bug?

CVE on `glob-parent` versions < 5.1.2. I'm unsure why this repository has not gotten a Dependabot alert on this (unless it is not enabled). `npm` is also alerting on this vulnerability.

### Current behavior:

CVE: https://github.com/advisories/GHSA-ww39-953v-wcq6

`glob-parent` is used in the following dependency tree:

```
@cypress/browserify-preprocessor > watchify > chokidar > glob-parent
```

### Desired behavior:

This CVE should ideally be addressed. I believe `chokidar` has released a new version 3.5.2 with the `glob-parent` dependency upgraded to 5.1.2. And the latest version of `watchify` should pick up `chokidar` 3.5.2. This repo has to update its `watchify` dependency.

### How to reproduce:

`npm audit` on this repository should alert on this. Otherwise, see the GitHub Advisories link above.

### Additional Info (images, stack traces, etc)

N/A

**Comment by pitgrap** [comment]
I get the same vulnerability report, but with a light different dependency tree.

```
@cypress/browserify-preprocessor > babel-plugin-add-module-exports > chokidar > glob-parent
```

**Comment by willrobb7** [comment]
Looks like renovate bot has submitted a PR to fix this! 
https://github.com/cypress-io/cypress-browserify-preprocessor/pull/84/commits/cf7ded9ce924182431624802c9107443e4904280

**Comment by MajorCaiger** [comment]
> I get the same vulnerability report, but with a different dependency tree.
> 
> ```
> @cypress/browserify-preprocessor > babel-plugin-add-module-exports > chokidar > glob-parent
> ```

`babel-plugin-add-module-exports` has an optional dependency on chokidar, you can run npm install with "--no-optional" to fix that (You may need to remove your package-lock and/or node_modules first though). However, you'll then get the reported dependency tree instead.

**Comment by binarybaba** [comment]
Are we even planning to accept renovate's PR?

**Comment by runswithd6s** [comment]
Running `npm install --no-optional` does not succeed in preventing `glob-parent 3.1.0` from being included, due to the dependency upon `babel-plugin-add-module-exports` -- which is unmaintained and hasn't seen a commit in GitHub in years.

**Comment by chrisbreiding** [comment]
Fixed in [v3.0.2](https://github.com/cypress-io/cypress-browserify-preprocessor/releases/tag/v3.0.2).
